### DISCUSSION TITLE
Extremely high bandwidth usage on a simple website

### DISCUSSION THREAD

**Discussion by evertjr** [vercel/community#462] [Help] [answered]
Hi, we've noticed very high outgoing bandwidth usage on our website https://www.embracon.com.br/

it's a simple Next.js page with mostly static content. On the Vercel admin portal it reports usage over 600GB outgoing traffic DAILY.

We are being billed for exceeded traffic and it's very concerning as the provided information isn't enough to track down the issue.
Can someone here from Vercel help?

**Comment by JTaylor0196** [comment, marked as answer]
@evertjr Hey there! Given that this is a billing concern and we may need to discuss personal data, please send a support request through to Vercel Support, and someone from the team will be able to better assist you there! -> https://vercel.com/support/request

**Reply by ckopecky** [reply]
Hi there Evert! I responded to Leonardo this morning not long after I signed on for the morning. . . Around 9am PST. I just added you to the thread so that you can schedule a meeting with your account executive to hopefully figure out what is going on!

**Reply by evertjr** [reply]
Hi! I talked to Leonardo and he didn’t get this response. Then I contacted Vercel Support on Twitter seeking help, the representative sent a test email to him and he received correctly. Now we’re waiting  a new response to figure out how we can solve this. We’re concerned because the bill is currently way over our budget. $3000 dollars in a month for this website doesn’t look right.

**Reply by evertjr** [reply]
Hi I checked my email and I saw the thread, we’ll  continue there.
